guide_drafts/backend_installation.md

```diff
@@ -0,0 +1,74 @@
+# Installing Backend Client Libraries
+
+Diesel supports SQLite, PostgreSQL, and MySQL as backend databases. By default, `diesel_cli` requires the client library of all three backends to be installed. If one is missing, then `cargo install diesel_cli` you will throw an error like:
+
+```
+note: ld: library not found for -lmysqlclient
+clang: error: linker command failed with exit code 1 (use -v to see invocation)
+```
+To install `diesel_cli` without all backends, specify `--no-default-features`. Use `--features` option to specify `postgres`, `sqlite`, and/or `mysql`. For example, to install with sqlite only, run:
```

Comment:
"Use [the] `--features` option"

Or maybe "`--features` flag"? These parameters are often referred to as "CLI flags"

Reply:
I checked the cargo man page and it's listed under `Options:`. I always think of a "CLI flag" as an option that doesn't take an argument. How about:
>Use cargo's `--features` option

Reply:
> I always think of a "CLI flag" as an option that doesn't take an argument

That's a good point! I hadn't thought about it before and that totally makes sense